python/tvm/relay/backend/graph_runtime_codegen.py

```diff
@@ -283,6 +283,12 @@ def visit_call(self, call):
     def visit_op(self, _):
         raise Exception("can not compile op in non-eta expanded form")
 
+    def visit_constructor(self, _):
```

Comment:
@MarisaKirisame @tqchen I am not familiar with this section of the codebase so I would need some pointers on what code should be generated for the ADT cases (tagging you because `git blame` showed you wrote most of this file)

Reply:
Alright, I guess

Reply:
Understood, I'll change the error message.